<launch>

  <node name="stereo_depth_estimation"
        pkg="stereo_image_sandbox" type="stereo_depth_estimation.py"
        output="screen" >
    <remap from="~left/image_rect" to="/left/image_rect_color" />
    <remap from="~left/camera_info" to="/left/camera_info" />
    <remap from="~right/image_rect" to="/right/image_rect_color" />
    <remap from="~right/camera_info" to="/right/camera_info" />
    <rosparam>
      approximate_sync: true
      slop: 1.0
    </rosparam>
  </node>

  <node name="depth_to_points"
        pkg="nodelet" type="nodelet"
        args="standalone depth_image_proc/point_cloud_xyzrgb">
    <param name="queue_size" value="100" />
    <remap from="rgb/camera_info" to="/left/camera_info" />
    <remap from="rgb/image_rect_color" to="/left/image_rect_color" />
    <remap from="depth_registered/image_rect" to="stereo_depth_estimation/output" />
    <remap from="depth_registered/points" to="/stereo/estimated_points" />
    <rosparam>
      approximate_sync: true
    </rosparam>
  </node>

</launch>
